src/ViewRequest.js

```diff
@@ -209,6 +212,11 @@ class ViewRequest extends React.Component {
     });
   }
 
+  craftLayerUrl(mode) {
```

Comment:
Duplicate of https://github.com/folio-org/ui-requests/pull/135?

Reply:
Merged.